ruby/README.md

```diff
@@ -68,13 +68,23 @@ To Build the JRuby extension, you will need:
 
 First switch to the desired platform with rbenv or RVM.
 
-Then install the required Ruby gems:
+Then install the required Ruby gems (MRI):
 
     $ gem install bundler
     $ bundle
 
+Then install the required Ruby gems (JRuby):
+
+    $ jgem install bundler
```

Comment:
Is `jgem` necessary even when using rvm? IIRC, I had managed to just `gem install bundler` while in an environment that had selected JRuby via rvm, but it's been a while.